apps/system/js/nfc_manager.js

```diff
@@ -37,11 +37,24 @@
    * @requires MozActivity
    * @requires NDEF
    * @requires NfcUtils
-   * @requires NfcHandoverManager
    */
-  var NfcManager = function() {};
+  var NfcManager = function() {
```

Comment:
var NfcManager = function(core) {
  this.core = core;
  this.nfc = core.nfc;
};